### ISSUE TITLE
heredoc bug. Prints an extra character after newline

### ISSUE THREAD

**Issue by sophiakoulen** [sophiakoulen/minishell#37]

**Comment by sophiakoulen** [comment]
```bash
minishell$ cat <<EOF
> hello world this is a hereoc
> herdoc heredoc heredoc
> EOF
hello world this is a hereoc
�herdoc heredoc heredoc
�minishell$
```

**Comment by sophiakoulen** [comment]
closed with 51f373a1a56b240c577a8c2047413302f97593f3
